interp/interp.go

```diff
@@ -83,6 +83,8 @@ type Runner struct {
 
 	dirStack []string
 
+	optState getopts // getopts
```

Comment:
Is the comment here useful?

Reply:
With struct named getopts probably not. Removed.